### ISSUE TITLE
Return geocoder service name in repsonse

### ISSUE THREAD

**Issue by z3to** [nchaulet/node-geocoder#173]
It would be great to have the service you have requested in the response object. For example:

``` javascript
[ {
    service: 'google',
    formattedAddress: '30 Av. des Champs-Élysées, 75008 Paris, France',
    latitude: 48.87001679999999,
    longitude: 2.3085181,
    extra: 
     { googlePlaceId: 'ChIJPc9VUMRv5kcRIE7VwbT8xM8',
       confidence: 1,
       premise: null,
       subpremise: null,
       neighborhood: null,
       establishment: null },
    administrativeLevels: 
     { level2long: 'Paris',
       level2short: 'Paris',
       level1long: 'Île-de-France',
       level1short: 'Île-de-France' },
    streetNumber: '30',
    streetName: 'Avenue des Champs-Élysées',
    city: 'Paris',
    country: 'France',
    countryCode: 'FR',
    zipcode: '75008' } ]
```

**Comment by nchaulet** [comment]
I think `provider` will be a better name for the field because it's the terms used in the documentation an internally. but great idea 👍

**Comment by nchaulet** [comment]
`provider` field is available in v3.11.0
